lib/index.js

```diff
@@ -22,11 +32,33 @@ class Client {
     this.fetch = (...args) => this.queue(fetch.bind(this, ...args))
   }
 
+  setModelsPath (path) {
+    this.modelsPath = path
+  }
+
+  setEntriesPath (path) {
+    this.entriesPath = path
+  }
+
   migrateContentTypes () {
     const models = fs.readdirSync(this.modelsPath).map((f) =>
       require(`${this.modelsPath}/${f}`)
     )
-    return Promise.all(models.map((m) => this.updateContentModel(m)))
+    return Promise.all(models.map((m) => this.updateContentType(m)))
+  }
+
+  migrateEntries () {
+    const entries = fs.readdirSync(this.entriesPath).map((f) =>
+      require(`${this.entriesPath}/${f}`)
+    )
+    return Promise.all(flatten(entries).map((e) => this.updateEntry(e)))
+  }
+
+  seedEntries () {
+    const entries = fs.readdirSync(this.entriesPath).map((f) =>
+      require(`${this.entriesPath}/${f}`)
+    )
+    return Promise.all(flatten(entries).map((e) => this.createEntry(e)))
```

Comment:
This looks great, much cleaner and more DRY 🙌 

Maybe just use a `path.join` rather than the manual string concat in the `readDataFromPath` method, otherwise this looks perfect